docs/production.md

```diff
@@ -1,14 +1,18 @@
 
 
+
+
 ## Production Environment Documentation
 
 The system is Dockerized into a number of components:
 
 * `nginx` - A reverse proxy that routes requests to the dashboard
 * `redis` - Job management as well as permanent variable storage
-* `mongo` - Storage of dockets, documents, and comments (in parallel with disk storage)
+* `mongo` - Storage of dockets, documents, comments and attachments(in parallel with disk storage)
```

Comment:
You may want to add a space after attachments

Reply:
Thanks Justin.